README.md

```diff
@@ -0,0 +1,47 @@
+# Cylc TextMate Grammar
+
+A TextMate bundle for working with [Cylc](www.cylc.org) workflow
```

Comment:
Without the protocol in front of the URL, in Markdown that gets resolved to a relative URL of the current git repo